src/components/About.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { Container, Row, Col } from 'react-bootstrap';
+import { Card } from 'react-bootstrap';
+import Avatar from 'react-avatar';
+import {FaFacebook}  from 'react-icons/fa';
+import {FaInstagram} from 'react-icons/fa';
+import {FaTwitter} from 'react-icons/fa'
```

Comment:
It looks like you have added a new package call "react-icons" but there are no changes to the package.json file. If you want to use this package the package.json file also needs to be committed to this pull request.